### DISCUSSION TITLE
SB500 bricked? Remotes for Tivo and Humax?

### DISCUSSION THREAD

**Discussion by Alimond52** [GerryDazoo/Slinger#550] [Q&A]
I have two SB 500 boxes and two ProHD boxes, and the later which have been unbricked several times. Now I have blocked all Internet traffic to these boxes so I finally hope they don't get bricked again.
I have not yet tested the SB500 on the Internet. But have any of you got an **SB500 bricked?**
I'm going abroad soon and would be happy if I could get Slinger up and running with remote controls.
Slinger streams well from all boxes.
But some thoughts:
1) It doesn't seem to be full HD resolution when I look at the stream from the SB500 HDMI output. In your Slingplayer I get **1920 / 540**. Is that the maximum?
2) Then I see that the bit rate is quite low = **2.5 Mbits**. When I streamed before 9N, the image quality was pretty good with speeds over 6 Mbits. And I got that easily from my network here and abroad, where I will be in a few days. Is there any way to increase the speed? I have set 8 Mbits in the config.ini file.
3) **Remote** controls? My sources are three STBs: Two old **Humax** and one **Tivo**.
I had two good remotes that worked perfectly before the 9N. Humax, I have **.bin and .spr files**. But since both ProHDs have been bricked, it has not been possible to upload those perfect remote controls anymore.
Is it the case that the remote controls you created for Slinger use correct IR codes from the SB boxes, but you don't get a true-to-life **Skin** = appearance on the virtual remote control?
4) **Remote** control options? I look at the names of the remote controls you created and several are **Tivo**. There have been many Tivos made over the years, but the remotes are pretty similar.
This is what my remote looks like. Of the Tivo remotes you created, which do you **recommend** for my Tivo box?
And which remote would fit my **Humax**. All mine have **HDD DVR**s. See the photo
5) How does it look on the **screen** with the Slinger and the Remote control box? Is it possible, for example, to have the video window full size and the remote control window is on top, and disappears **automatically** after a few seconds of mouse inactivity? So was the way the old Slingplayers worked?
6) I understand that you can have different **profiles** for each SB. What is the easiest way to switch between these three SBs when I want to watch via the new Slingplayer?
Sorry if I'm asking questions that have already been addressed. But it takes time to search for answers here on the forum.
Many **thanks** Gerry and the rest for all good job!
![Remotes](https://user-images.githubusercontent.com/123761450/222155955-3c214b16-e542-40ad-9b50-776ee52fea61.jpg)
![Slingplayer PC](https://user-images.githubusercontent.com/123761450/222156029-05f0b08e-5a60-4f58-a7af-dbd4078d2670.JPG)

**Comment by GerryDazoo** [comment]
So many questions.
You wrote "I have blocked all Internet traffic to these boxes" you need to block Internet traffic FROM the Slingbox not TO the Slingbox.

You said "In your SlingPlayer" do you mean SlingerPlayer? All it does is play the stream sent to it from Slinger. It has no control over what is sent to it. If you're getting 1920x540 then that's what the slingbox is sending. Without seeing your config.ini file it's not possible to tell what might be going on but I suspect your using resolution 15 not 16. 
The bit rate is whatever the Slingbox is sending. It's also a function of what you're viewing.
I didn't create those TIVO remotes I have no recommendations except to try them and see if they work to your satisfaction.
You should be able to configure the remote when un-bricking the Pro.
5) No, that capability is not planned for SlingerPlayer
6) Again I assume you mean SlingerPlayer not SlingPlayer. You type the URL for the Slinger config into the URL field and press play. If you've already connected once they are saved over sessions and you can just click on the drop down list to see all your recent connections.
You show pictures of SlingPLayer not SlingerPLayer so that has also confused me. 
Please be advised that the Sling supplied "SlingPlayer" will NOT work remotely. You will need to use some other software to view your video stream.

**Reply by Alimond52** [reply]
Thanks Gerry. I block both incoming and outgoing traffic. Yes I mean Slingerplayer you made. 
2) I will check settings later. 
3) Bitrate 2,5 Mbits is very strange, it is the speed inside my LAN to Slingerplayer. Usullay before N9 the speed was 8 Mbits on Slingplayer.
4) I cannot create the remote with correct skin since old Slingplayer cannot connect to Set Up utility due non-support flashplayer.
6) It sounds very good. I will test.
Yes the picture of old Slingplayer just want to show how my remote interact with full screen video before N9.
Your last sentance I am unsure what you mean: **"Please be advised that the Sling supplied "SlingPlayer" will NOT work remotely. You will need to use some other software to view your video stream**."

**Reply by GerryDazoo** [reply]
4) You must use SlingPlayer V1.5 not the 2.0 flash based version.
6) It appears your definition of fullscreen and mine are different.
Any Sling supplied software like SlingPlayer inn your picture will not let you view your slingbox remotely.

**Reply by Alimond52** [reply]
Thanks Gerry. I have used V1.5, for unbricking. But cannot load the correct skin for my Humax remote.
Actually it was possible ot use Sling Slingplayer remotely before I do more blocking of ports in Router. Since I use Direct IP connection in the version 2 of SlingPlayer. After more portblocking it still work inside my LAN now, but not always.

**Reply by Alimond52** [reply]
Going abroad in a few days and hoping to enjoy Slinger for the first time.
Some problems are solved, thanks for the help!
1) Full HD was value no. 16 as you told me. I had entered "15" as it was stated to be the maximum resolution.
2) The video stream is good.
Your SlingerPlayer is simple and good too. Good job!
4) It took many hours to get a properly working **remote** control for my old Tivo box. The last one any of you made worked with almost all features, plus it was nicely made "skin" similar to my physical remote. But some functions are on the "wrong" buttons. But I can control the most important thing.
Then it remains to get a **remote** that works with my old **Humax** box. The base remote control you made can be used to control most things except the most important functions: There are a number of Custom codes 10-25 in the base remote control. Most important being able to access the **HDD recordings (DVR Media)**. That button has value number “**14**”. And then button number "**15**", which is marked on my physical remote with **Opt+**, to remove a recording reservation.
Facts about the STB boxes: Humax HDPVR 1000C, The remote control is marked: Humax RM-301.
Files for these are and attached here (not possible):
V2010_PL.bin
Humax_RM-301.spr
Codes: Humax:
Keycodes= [1, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 21, 24, 25, 26, 27, 28, 29, 30 , 31, 33, 35, 36, 38, 39, 40, 41, 42, 46, 62, 63, 64, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76 , 77]
Here is the complete Function list
Custom 10: Audio
Custom 11: List
Custom 12: Bookmarks
Custom 13: Edit
**Custom 14: Media**
**Custom 15: Opt+**
Custom 16: Source
Custom 17: subtitles
Custom 18: teletext
Custom 19: TV/radio
Custom 20: v-shape
Custom 21: wide
Custom 22: Ed
Custom 23: Green
Custom 24: Yellow
Custom 25: Blue
7) The **Remote.txt** file. I suppose I can **edit** it in Text editor? Some buttons/functions I'm not used to. What do they mean? **"Day" "Pg" "OnDemand"**
6) I have a total of four different SBs, which control three STBs, two of which are the same Humax and then one Tivo.
I will use your excellent Slinger player abroad. How do you set up the **easiest** way to be able to **switch** between these four SB boxes. Config.ini? Links on desktop? I will see everything from a PC, Windows 10 connected to a TV screen.

**Reply by gbjbasdw** [reply]
Not sure if this helps but I am currently away from home using Slinger Server V4 and SlingerPlayer connected to a ProHd in the UK with a Virginmedia Tivo V6 and a Humax Freesat as input sources. I have two remote.txt files - one for each STB. I use my mobile phone's web browser as the remote control by going to the http://172.17.0.10:64000/Remote/ProHD_Src1 for example. The IP address is what I am on over here the 64000 is the opened port and the" remote" is for the Pro_HD input source 1 which is the composite input connected to the the Tivo V6 STB. For the Humax box the end of the string is ProHD_Src3 ( composite input connected to the Humax box) or ProHD_Src0 for the ProHD internal tuner.

**Reply by Alimond52** [reply]
Thank you so much!
I have got the Slinger Player working OK today with one of my SB500, and a Tivo remote got from here:
https://github.com/GerryDazoo/Slinger/discussions/538#discussioncomment-5216465
But still problem with my Humax to the other SB500, I cannot get the remote tor working. It works Ok when sending to SB ProHD. I am using here the standard **remote.txt** on this site.
What remote file do you use to your Humax box? Have you modified it yourself?
Is it difficult to **modify** standard remote.txt file for other code for buttons?
